FastSurferCNN/README.md

```diff
@@ -54,12 +61,12 @@ The output will be saved in ./data/subject1/aparc.DKTatlas+aseg.deep.mgz.
 To run the network on all subjects MRI-volumes in ./data, change into the *FastSurferCNN* directory and run the following command: 
 
 ```
-python3 eval.py --i_dir ../data \
---o_dir ../data \
---log temp_Competitive.log \
---network_sagittal_path ../checkpoints/Sagittal_Weights_FastSurferCNN/ckpts/Epoch_30_training_state.pkl \
---network_coronal_path ../checkpoints/Coronal_Weights_FastSurferCNN/ckpts/Epoch_30_training_state.pkl \
---network_axial_path ../checkpoints/Axial_Weights_FastSurferCNN/ckpts/Epoch_30_training_state.pkl 
+python3 eval.py --in_dir ../data \
+--sd ../data \
+--seg_log temp_Competitive.log \
+--ckpt_sag ../checkpoints/Sagittal_Weights_FastSurferCNN/ckpts/Epoch_30_training_state.pkl \
+--ckpt_cor ../checkpoints/Coronal_Weights_FastSurferCNN/ckpts/Epoch_30_training_state.pkl \
+--ckpt_ax ../checkpoints/Axial_Weights_FastSurferCNN/ckpts/Epoch_30_training_state.pkl 
 ```
 
 The output will be stored in ./data/subjectX/aparc.DKTatlas+aseg.deep.mgz.
```

Comment:
Here, it should be mentioned, that the log-file created here will have all outputs (all subjects logged into one) and that is also possible to leave the log-file undefined in which case everything is written to stdout.